scripts/prepare_doc_publication.py

```diff
@@ -62,6 +62,9 @@ def parse_arguments():
     'objectives.md': 'Short-term Objectives',
     'tensorflow_integrations.md': 'TensorFlow Integrations',
     'e2e_benchmarking.md': 'Benchmarking TensorFlow with IREE and TFLite',
+    'profiling_cpu_events.md': 'CPU events',
+    'profiling_with_tracy.md': 'Tracy',
+    'profiling_vulkan_gpu.md': 'Vulkan GPU',
```

Comment:
put "Profiling" in these titles

```suggestion
    'profiling_cpu_events.md': 'Profiling CPU events',
    'profiling_with_tracy.md': 'Profiling with Tracy',
    'profiling_vulkan_gpu.md': 'Profiling Vulkan GPUs',
```

Reply:
done